lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -208,7 +228,7 @@ rule RootWebHome {
 	   	title <- 'Home',
 		parent <- 'Main.WebHome',
 		wiki <- 'xwiki',
-		space <- thisModule.getModelSetId(),
+		space <- thisModule.modelSet.name,
```

Comment:
You're using the `name` but I think the modelset ID is one of the attribute of this instance.  Look [here](https://github.com/MDEGroup/learnpad/blob/VilniusDecisionV1/lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/model/ado4f16a6bb-9318-4908-84a7-c2d135253dc9.xml#L5674). Does what I say make any sense?